src/main/java/com/softserve/itacademy/controller/GroupController.java

```diff
@@ -0,0 +1,40 @@
+package com.softserve.itacademy.controller;
+
+import com.softserve.itacademy.dto.GroupDto;
+import com.softserve.itacademy.service.GroupService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@RequestMapping("/api/groups")
+public class GroupController {
+
+    private final GroupService groupService;
+
+    public GroupController(GroupService groupService) {
+        this.groupService = groupService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<GroupDto>> findAll() {
+        return new ResponseEntity<>(groupService.findAll(), HttpStatus.OK);
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Void> delete(@PathVariable Integer id) {
+        groupService.delete(id);
+        return new ResponseEntity<>(OK);
+    }
+
+    @DeleteMapping("/{id}/disabled")
```

Comment:
This is not correctly designed endpoint.
Use PATCH instead to update one filed. Why did you use DELETE?